yarn-project/noir-contracts/src/contracts/child_contract/src/main.nr

```diff
@@ -28,8 +28,8 @@ contract Child {
     // Nested public functions always get called with MAX_ARGS, since we don't have the ABI available
     // during execution time to know how many args are expected, hence the _padding argument. We should
     // be able to remove it when we migrate to brillig.
-    open fn pubValue(_inputs: PublicContextInputs, base_value: Field, _padding: [Field; abi::MAX_ARGS - 1]) -> pub Field {
-        base_value + 42
+    open fn pubValue(inputs: PublicContextInputs, base_value: Field, _padding: [Field; abi::MAX_ARGS - 1]) -> pub Field {
```

Comment:
maybe we should make this a seperate test/

Reply:
Were thinking that it was fine as this since we were anyway just returning random values, so might as well return the values from the struct.